rtv/__main__.py

```diff
@@ -24,7 +24,6 @@ def load_config():
     Search for a configuration file at the location ~/.rtv and attempt to load
     saved settings for things like the username and password.
     """
-
     config = configparser.ConfigParser()
```

Comment:
I chose `parser` for instance of `configparser.ConfigParser`, this is more accurate and avoid lot of `conf*` variables